config/locales/en.yml

```diff
@@ -68,7 +68,9 @@ en:
     # flash.notice is only used by Devise and is treated as important.
     alert:
       title: Important
-
+      user_deactivated: Unable to deactivate user. There must be at least two users who can manage others
+    error:
+      title: Error in application
```

Comment:
This `error.title` translation was required because without it exceptions bubbling to the top of the application failed again due to an expected HTML error page translation